src/components/styles.js

```diff
@@ -29,7 +29,7 @@ export const ClauseHeader = styled.div`
   justify-self: start;
   margin: 6px 0;
   padding: 3px;
-  color: ${props => props.headercolor || '#939EBA'};
+  color: ${props => props.headercolor || '#63779C'};
```

Comment:
The background here is gradient from `#FFFFFF` to `#ECF0FA` , so I would think something darker like `#5B6E8F` would be necessary.

Reply:
Let's use #505D7E

Reply:
#5B6E8F is indeed giving more contrast. I checked it w.r.t #FFFFFF (given as background color by wave extension). I will change this.

Reply:
#5B6E8F is giving me 5.15 contrast where as #505D7E is giving contrast 6.54

Reply:
That sounds good.

Reply:
let's use #505D7E please